app/models/manager_refresh/save_collection/base.rb

```diff
@@ -2,16 +2,16 @@ module ManagerRefresh::SaveCollection
   class Base
     class << self
       def save_inventory_object_inventory(ems, inventory_collection)
-        _log.info("Synchronizing #{ems.name} collection #{inventory_collection} of size #{inventory_collection.size} to"\
-                " the database")
+        _log.info("SYNCHRONIZING collection #{inventory_collection} of size #{inventory_collection.size} to"\
```

Comment:
:+1: can you change https://github.com/ManageIQ/manageiq/pull/15707/files#diff-1f612b8c6334aa147103033d326ab215R9 to match then?